server/common/zk/src/main/java/org/kaaproject/kaa/server/common/zk/gen/LoadInfo.java

```diff
@@ -0,0 +1,141 @@
+/**
+ * Autogenerated by Avro
+ * 
+ * DO NOT EDIT DIRECTLY
+ */
+package org.kaaproject.kaa.server.common.zk.gen;  
+@SuppressWarnings("all")
+@org.apache.avro.specific.AvroGenerated
+public class LoadInfo extends org.apache.avro.specific.SpecificRecordBase implements org.apache.avro.specific.SpecificRecord {
+  public static final org.apache.avro.Schema SCHEMA$ = new org.apache.avro.Schema.Parser().parse("{\"type\":\"record\",\"name\":\"LoadInfo\",\"namespace\":\"org.kaaproject.kaa.server.common.zk.gen\",\"fields\":[{\"name\":\"loadIndex\",\"type\":\"int\"}]}");
+  public static org.apache.avro.Schema getClassSchema() { return SCHEMA$; }
+   private int loadIndex;
+
+  /**
+   * Default constructor.  Note that this does not initialize fields
```

Comment:
A default constructor. Note that it does not initialize fields to their default values from the schema. If that is desired, use {@link \#newBuilder()}.